src/scripts/main.js

```diff
@@ -357,4 +357,26 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
Ensure that the table element exists in the DOM and is correctly selected. If the table is not present, this line will throw an error.